cmd/edgectl/aes_install_errors.go

```diff
@@ -141,62 +158,85 @@ The installer will quit to avoid corrupting an existing installation of AES.`
-		Err:    err,
+		Report:       "fail_install_crds",
+		ShortMessage: "An error occurred while applying Kubernetes manifests",
+		Message:      fmt.Sprintf("Visit %v for information about getting in touch with us about this error.", url),
+		URL:          url,
+		Err:          err,
 	}
 }
 
 // 90-second timeout on waiting for aes-crd.yaml manifests to be established
 func (i *Installer) WaitCRDsError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/wait-crds"
+
 	return Result{
-		Report: "fail_wait_crds",
-		URL:    "https://www.getambassador.io/docs/latest/topics/install/help/wait-crds",
-		Err:    err,
+		Report:       "fail_wait_crds",
+		ShortMessage: "An error occurred while applying Kubernetes manifests",
+		Message:      fmt.Sprintf("Visit %v for information about getting in touch with us about this error.", url),
+		URL:          url,
+		Err:          err,
 	}
 }
 
 // Unable to kubectl apply the aes.yaml manifests
 func (i *Installer) InstallAESError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/install-aes"
+
 	return Result{
-		Report: "fail_install_aes",
-		URL:    "https://www.getambassador.io/docs/latest/topics/install/help/install-aes",
-		Err:    err,
+		Report:       "fail_install_aes",
+		ShortMessage: "An error occurred while applying Kubernetes manifests",
+		Message:      fmt.Sprintf("Visit %v for information about getting in touch with us about this error.", url),
+		URL:          url,
+		Err:          err,
 	}
 }
 
 //90-second timeout on waiting for aes.yaml manifests to be deployed and available
 func (i *Installer) WaitForAESError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/wait-for-aes"
+
 	return Result{
-		Report: "fail_wait_aes",
-		URL:    "https://www.getambassador.io/docs/latest/topics/install/help/wait-for-aes",
-		Err:    err,
+		Report:       "fail_wait_aes",
+		ShortMessage: "An error occurred while applying Kubernetes manifests",
+		Message:      fmt.Sprintf("Visit %v for information about getting in touch with us about this error.", url),
+		URL:          url,
+		Err:          err,
 	}
 }
 
 // Unable to get the AES Install ID via kubectl exec to ask for the pod ID
 func (i *Installer) AESPodStartupError(err error) Result {
+	url := "https://www.getambassador.io/docs/latest/topics/install/help/aes-pod-startup"
+
 	return Result{
-		Report: "fail_pod_timeout",
-		URL:    "https://www.getambassador.io/docs/latest/topics/install/help/aes-pod-startup",
-		Err:    err,
+		Report:       "fail_pod_timeout",
+		ShortMessage: "The installer was unable to talk to your AES pod",
+		Message:      fmt.Sprintf("The installer failed to communicate with an AES Pod in your cluster to validate the installation.  Visit %v for a more detailed explanation and step-by-step instructions about resolving this problem.", url),
```

Comment:
Message should start with "Visit ..." like all the others; move the specific message here to the short message (or remove it, since the short message seems OK already).

Reply:
![new AESPodStartupError](https://user-images.githubusercontent.com/635244/78847453-ae6a3100-79c3-11ea-836b-2f61fd11aa68.png)

Reply:
is this OK?  That's what it looks like now.

Reply:
We should remove the "The installer failed to communicate with an AES Pod in your cluster to validate the installation." sentence from the message and keep it to "Visit ..." just like the rest. (This was the only one that didn't follow the consistent pattern.)